pkg/lib/oauth/handler/handler_authz.go

```diff
@@ -809,6 +814,7 @@ func (h *AuthorizationHandler) doHandleConsentRequest(
 			IDTokenHintSID:       idTokenHintSID,
 			Cookies:              []*http.Cookie{},
 			GrantAuthz:           opts.GrantAuthz,
+			IdentitySpecs:        opts.ConsentRequest.OAuthSessionEntry.T.IdentitySpecs,
```

Comment:
I think `AuthInfoEntry.T` is a better place to store `IdentitySpecs`, because it is something produced **after** the authentication. And `OAuthSessionEntry` seems to be storing something related to the request, to things that used during the authentication.